doc/api/worker_threads.md

```diff
@@ -654,7 +658,9 @@ changes:
 * `filename` {string|URL} The path to the Worker’s main script or module. Must
   be either an absolute path or a relative path (i.e. relative to the
   current working directory) starting with `./` or `../`, or a WHATWG `URL`
-  object using `file:` protocol.
+  object using `file:` or `data:` protocol.
+  When using a [`data:` URL][], the JavaScript code will be interpreted as an
```

Comment:
I think this is only partially true. It currently sounds like `data:text/plain,...` would be interpreted as an ECMAScript module but it isn't, afaict. Maybe "interpreted based on the mime type" with a link to the module docs on data URL support? Alternatively: "interpreted using the ECMAScript module loader" with a similar link?